docker/rp-complete.dockerfile

```diff
@@ -16,38 +16,43 @@
 #     docker kill rp_test
```

Comment:
I don't appreciate the difference between the two different dockerfiles - aren't they basically the same?

Reply:
One has MongoDB and is built on the `mongo:bionic` image. The other does not have mongodb, but runs an sshd and is meant to be composed with a standalone mongodb container. I find it easier to use the composed stack interactively, but the combined container is maybe marginally easier for automation or less adventurous docker users.